test/minter-suite-minters/DA/MinterDAExp/MinterDAExp.common.ts

```diff
@@ -193,6 +193,77 @@ export const MinterDAExp_Common = async () => {
     });
   });
 
+  describe("projectAuctionParameters", async function () {
+    it("returns expected populated values", async function () {
+      const auctionParams = await this.minter.projectAuctionParameters(
+        this.projectZero
+      );
+      expect(auctionParams.timestampStart).to.be.equal(
+        this.startTime + this.auctionStartTimeOffset
+      );
+      expect(auctionParams.priceDecayHalfLifeSeconds).to.be.equal(
+        this.defaultHalfLife
+      );
+      expect(auctionParams.startPrice).to.be.equal(this.startingPrice);
+      expect(auctionParams.basePrice).to.be.equal(this.basePrice);
+    });
+
+    it("returns expected initial values", async function () {
+      const auctionParams = await this.minter
+        .connect(this.accounts.deployer)
+        .projectAuctionParameters(this.projectOne);
+      expect(auctionParams.timestampStart).to.be.equal(0);
+      expect(auctionParams.priceDecayHalfLifeSeconds).to.be.equal(0);
+      expect(auctionParams.startPrice).to.be.equal(0);
+      expect(auctionParams.basePrice).to.be.equal(0);
```

Comment:
Nit: This is a pretty regularly repeated block across our tests, could be worth abstracting into a convenience method (can be done in a follow up PR).

Reply:
updated! 🙏 thanks!